src/platform/ESP32/NetworkCommissioningDriver.cpp

```diff
@@ -147,6 +147,11 @@ CHIP_ERROR ESPWiFiDriver::CommitConfiguration()
 CHIP_ERROR ESPWiFiDriver::RevertConfiguration()
 {
     mStagingNetwork = mSavedNetwork;
+    if (!GetEnabled())
```

Comment:
Please split out the server-side changes into a separate PR from the controller/client changes.

Possibly separate PRs for ESP32 and OpenThread, since they need to be reviewed by different people.

Reply:
Ok, will add PRs for server-side changes.